components/places/ffi/src/lib.rs

```diff
@@ -480,17 +480,17 @@ pub extern "C" fn places_accept_result(
 #[no_mangle]
 pub extern "C" fn places_reset(handle: u64, error: &mut ExternError) {
     log::debug!("places_reset");
-    CONNECTIONS.call_with_result(error, handle, |conn| -> places::Result<_> {
-        storage::history::history_sync::reset(conn)?;
+    APIS.call_with_result(error, handle, |api| -> places::Result<_> {
+        api.reset_history()?;
```

Comment:
We call `reset_history` instead of `storage::history::history_sync::reset`, does that look ok to you @linacambridge ? It seems like it roughly does the same thing.

Reply:
I'd kind of like @linacambridge to weigh in here before landing

Reply:
Yup it does